content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -112,28 +114,30 @@ The upgrade workflow at high level is the following:
     [upgrade/config] Reading configuration from the cluster...
     [upgrade/config] FYI: You can look at this config file with 'kubectl -n kube-system get cm kubeadm-config -oyaml'
     [preflight] Running pre-flight checks.
-    [upgrade] Making sure the cluster is healthy:
+    [upgrade] Running cluster health checks
     [upgrade] Fetching available versions to upgrade to
-    [upgrade/versions] Cluster version: v1.16.0
-    [upgrade/versions] kubeadm version: v1.17.0
+    [upgrade/versions] Cluster version: v1.17.3
+    [upgrade/versions] kubeadm version: v1.18.0
+    [upgrade/versions] Latest stable version: v1.18.0
+    [upgrade/versions] Latest version in the v1.17 series: v1.18.0
 
     Components that must be upgraded manually after you have upgraded the control plane with 'kubeadm upgrade apply':
-    COMPONENT   CURRENT       AVAILABLE
-    Kubelet     1 x v1.16.0   v1.17.0
+    COMPONENT   CURRENT             AVAILABLE
+    Kubelet     1 x v1.17.3   v1.18.0
 
-    Upgrade to the latest version in the v1.16 series:
+    Upgrade to the latest version in the v1.17 series:
 
     COMPONENT            CURRENT   AVAILABLE
-    API Server           v1.16.0   v1.17.0
-    Controller Manager   v1.16.0   v1.17.0
-    Scheduler            v1.16.0   v1.17.0
-    Kube Proxy           v1.16.0   v1.17.0
-    CoreDNS              1.6.2     1.6.5
-    Etcd                 3.3.15    3.4.3-0
+    API Server           v1.17.3   v1.18.0
+    Controller Manager   v1.17.3   v1.18.0
+    Scheduler            v1.17.3   v1.18.0
+    Kube Proxy           v1.17.3   v1.18.0
+    CoreDNS              1.6.5     1.6.7
+    Etcd                 3.4.3     3.4.3-0
 
     You can now apply the upgrade by executing the following command:
 
-        kubeadm upgrade apply v1.17.0
+        kubeadm upgrade apply v1.18.0 --allow-experimental-upgrades
```

Comment:
@neolit123 is the `--allow-experimental-upgrades` intended? Why's it now included?

Reply:
this was part of my setup to get the output.
good catch, will remove.